pkg/controller/volume/attachdetach/attach_detach_controller.go

```diff
@@ -900,7 +897,7 @@ func (adc *attachDetachController) addNodeToDswp(node *v1.Node, nodeName types.N
 		keepTerminatedPodVolumes := false
 
 		if t, ok := node.Annotations[volumeutil.KeepTerminatedPodVolumesAnnotation]; ok {
-			keepTerminatedPodVolumes = (t == "true")
+			keepTerminatedPodVolumes = t == "true"
```

Comment:
Although the change is correct, I personally prefer readability over correctness, your proposed change removes that aspect. Please revert back to original.

Reply:
done